packages/sku/src/utils/polyfillRegistry.ts

```diff
@@ -0,0 +1,47 @@
+export interface PolyfillRegistryEntry {
+  reason: string;
+  docsUrl?: string;
+}
+
+export type PolyfillRegistry = Record<string, PolyfillRegistryEntry>;
+
+/**
+ * Registry of polyfills that are unnecessary for sku's supported browsers
+ */
+export const POLYFILL_REGISTRY: PolyfillRegistry = {
+  'core-js': {
+    reason:
+      'Polyfills many features that may be unnecessary. Consider removing or using targeted polyfills for specific APIs',
+    docsUrl: 'https://github.com/zloirock/core-js#usage',
+  },
+  '@babel/polyfill': {
+    reason: 'Deprecated package',
+    docsUrl: 'https://babeljs.io/docs/en/babel-polyfill',
+  },
+  'babel/polyfill': {
+    reason: 'Deprecated package',
+    docsUrl: 'https://babeljs.io/docs/en/babel-polyfill',
+  },
+  'regenerator-runtime': {
+    reason:
+      'async/await is well established and works across many devices and browser versions',
```

Comment:
Can we use this similar messaging for the promise, fetch and intersection observer polyfills too? In fact maybe we should just copy the message from MDN. I like that it gives a date because it really drives home how old and useless these polyfills are.

```suggestion
      'async/await is well established and works across many devices and browser versions. It has been available across browsers since April 2017.',
```

[Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) were baseline in July 2015, fetch and [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) were baseline in April 2017, and [intersection observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) was baseline in March 2019.